packages/typespec-client-generator-core/src/lib.ts

```diff
@@ -3,25 +3,13 @@ import { createTypeSpecLibrary, paramMessage } from "@typespec/compiler";
 export const $lib = createTypeSpecLibrary({
   name: "@azure-tools/typespec-client-generator-core",
   diagnostics: {
-    "client-name": {
-      severity: "warning",
-      messages: {
-        default: paramMessage`Client name "${"name"}" must end with Client. Use @client({name: "...Client"}`,
-      },
-    },
     "client-service": {
       severity: "warning",
       messages: {
         default: paramMessage`Client "${"name"}" is not inside a service namespace. Use @client({service: MyServiceNS}`,
       },
     },
-    "unknown-client-format": {
-      severity: "error",
-      messages: {
-        default: paramMessage`Client format "${"format"}" is unknown. Known values are "${"knownValues"}"`,
-      },
-    },
-    "incorrect-client-format": {
+    "invalid-client-format": {
```

Comment:
Is this actually used?

Reply:
it's used when we're checking the input of [`@clientFormat`](https://github.com/Azure/typespec-azure/blob/main/packages/typespec-client-generator-core/src/decorators.ts#L793). Is this valid?

Reply:
I don't see what it is used, is it still there? the file you link doesn't have reference to that code

Reply:
I don't see it used in main anymore, could it have been removed